docs/entities/blocks/block.schema.json

```diff
@@ -72,6 +88,10 @@
         "type": "string",
         "description": "Microblock hash"
       }
+    },
+    "total_execution_cost": {
```

Comment:
I think this should have each of the individual execution cost units (`read_count`, `read_length`, `runtime`, `write_count`, `write_length`).